packages/hoprd/src/api/v2/paths/channel/tickets.ts

```diff
@@ -0,0 +1,78 @@
+import Hopr from '@hoprnet/hopr-core'
+import { Operation } from 'express-openapi'
+import { STATUS_CODES } from '../../'
+
+export const getTickets = async (node: Hopr) => {
+  const stats = await node.getTicketStatistics()
+
+  return {
+    pending: stats.pending,
+    unredeemed: stats.unredeemed,
+    unredeemedValue: stats.unredeemedValue.toFormattedString(),
```

Comment:
we should be sending over the smallest unit, this gives it back in formatted `10 HOPR`

Reply:
Agreed, the API is meant to be consumed by apps. Thus, these should be numbers, not formatted strings.